src/BlockTypes/Checkout.php

```diff
@@ -264,7 +264,7 @@ function() {
 
 		// Hydrate the following data depending on admin or frontend context.
 		if ( $is_block_editor && ! $this->asset_data_registry->exists( 'shippingMethodsExist' ) ) {
-			$methods_exist = wc_get_shipping_method_count( false, true ) > 0;
+			$methods_exist = wc_get_shipping_method_count( true, true ) > 0;
```

Comment:
the first value checks if we're going to include legacy methods (Local Pickup is considered legacy), the second value is if we're only including enabled values, this is kept the same.